src/app/Mappings/Network/NetworkMappingsPage.tsx

```diff
@@ -16,12 +16,23 @@ import { PlusCircleIcon } from '@patternfly/react-icons';
 import { INetworkMapping, MappingType } from '../types';
 import MappingsTable from '../components/MappingsTable';
 import AddEditMappingModal from '../components/AddEditMappingModal';
+import { fetchMockStorage } from '../mocks/helpers';
 
 // TODO replace these with real state e.g. from redux
 const isFetchingInitialNetworkMappings = false; // Fetching for the first time, not polling
-const networkMappings: INetworkMapping[] = [];
+// const networkMappings: INetworkMapping[] = [];
 
 const NetworkMappingsPage: React.FunctionComponent = () => {
+  const [networkMappings, setNetworkMappings] = React.useState([]);
```

Comment:
Let's throw a comment here too saying this is temporary, so we don't end up duplicating redux state here in react land